.npmignore

```diff
@@ -0,0 +1,12 @@
+## Directories
+examples/
+build/
```

Comment:
Wait, so we're ignoring the `build` directory? That doesn't seem right. Then users will have to build the library before they use it. IR files are tiny compared to the audio files in `examples`, so I think it's reasonable to keep the build directory.